git-features/Cargo.toml

```diff
@@ -20,14 +20,17 @@ progress = ["prodash"]
 
 ## If set, and with the `parallel` feature set, walkdir iterators will be single-threaded.
 ## This feature exists to avoid [certain side-effects](https://github.com/starship/starship/issues/4251) of rayon threadpool configuration with `jwalk`.
-fs-walkdir-single-threaded = []
+fs-walkdir-parallel = ["parallel", "jwalk" ]
 
 ## Use scoped threads and channels to parallelize common workloads on multiple objects. If enabled, it is used everywhere
 ## where it makes sense.
 ## As caches are likely to be used and instantiated per thread, more memory will be used on top of the costs for threads.
 ## The `threading` module will contain thread-safe primitives for shared ownership and mutation, otherwise these will be their single threaded counterparts.
 ## This way, single-threaded applications don't have to pay for threaded primitives.
-parallel = ["crossbeam-utils", "crossbeam-channel", "num_cpus", "jwalk", "parking_lot"]
+parallel = ["crossbeam-utils",
```

Comment:
Did you do this by hand or is it a tool? In case of the latter, I'd love to know :D. Currently I have no auto-formatter for toml files and am definitely not willing ever format files again 😁.

Reply:
Oh I didn't even notice I have autoformat  turned on for toml files.
The formatting is done by the [tablo lsp](https://github.com/tamasfe/taplo). I use it with helix but it also has a vscode extension and a CLI (to run in CI for example).

It's really nice lsp for toml files because it picks up the json schema so you get autocomoletions for most things (that don't require querying crates.io)

Reply:
Thanks a lot! `cargo install taplo-cli --locked` and `taplo fmt Cargo.toml` should be enough to get started and experiment.